services/common/src/redux/reducers/mineReducer.ts

```diff
@@ -95,6 +95,8 @@ const mineReducerObject = {
 };
 
 export const getMines = (state: RootState): ItemMap<IMine> => state[MINES].mines as ItemMap<IMine>;
+export const getMineById = (state: RootState, mineGuid: string): IMine =>
+  state[MINES]?.mines[mineGuid] as IMine;
```

Comment:
I do wonder why we never did this before

Reply:
Yeah.  I know I've wanted it before!  Nice!